demo/admin/src/lang.ts

```diff
@@ -15,7 +15,7 @@ const cometDemoMessages = {
     de: comet_demo_messages_de,
 };
 
-export const getMessages = (): ResolvedIntlConfig["messages"] => {
+export const getMessages = (language: "de" | "en"): ResolvedIntlConfig["messages"] => {
```

Comment:
Since we're already importing German messages, can we use the argument in this function?